ios/Demo Boutique/Demo Boutique/ProductDetailViewController.swift

```diff
@@ -23,11 +24,27 @@ class ProductDetailViewController: UIViewController {
         let product = self.product!
         self.nameLabel.text = product.name
         self.priceLabel.text = formatter.string(from: NSNumber(value: product.price))!
+             + " (plus $" + calculateShipping() + " shipping)"
         self.productImage.sd_setImage(with: product.image)
         self.descriptionTextView.text = product.description
 
         self.navigationItem.title = product.name
     }
+
+     // Here is an example of manual instrumentation
+     func calculateShipping() -> String {
+         let tracer = OpenTelemetry.instance.tracerProvider.get(instrumentationName: "manual_example", instrumentationVersion: "1.0")
```

Comment:
does the swift API require providing a version? If not, we should probably leave it out, as it will just confuse people.

Reply:
Yes, it does.